.gitlab-ci.yml

```diff
@@ -12,14 +12,15 @@ before_script:
 
 stages:
   - test
-  - build
   - publish
+  - trigger
 
 test:
   stage: test
   script:
     - make extracheck
     - make coverage
+    - make build
```

Comment:
`make build` is still here. Did you remove `V=1` by mistake?

Reply:
I miss-read your original comment. Good eye